lib/HighlightsFeed.jsm

```diff
@@ -228,34 +228,6 @@ this.HighlightsFeed = class HighlightsFeed {
     });
   }
 
-  /**
-   * Deletes an item from a user's saved to Pocket feed and then refreshes highlights
-   * @param {int} itemID
-   *  The unique ID given by Pocket for that item; used to look the item up when deleting
-   */
-  async deleteFromPocket(itemID) {
-    try {
-      await NewTabUtils.activityStreamLinks.deletePocketEntry(itemID);
-      this.fetchHighlights({broadcast: true});
```

Comment:
The old code would wait for the entry to be deleted before refreshing. But perhaps that ordering doesn't matter right now?